### DISCUSSION TITLE
Warpinator feature request, create shared folder

### DISCUSSION THREAD

**Discussion by spsf64** [linuxmint/discussions#302] [Ideas]
Is it possible to implement an option to create a shared folder so other workstations running warpinator could receive/download and send files without any interaction from the "server" computer?
This would be a "samba killer" feature!
TIA

**Comment by 100savage** [comment]
You could set up a share on a server and mount that share on your workstations. Just saying that this can be done right now without any changes to warpinator.

**Reply by spsf64** [reply]
Wouldn't this require samba? Which, for me,  is a pain to configure...

**Reply by 100savage** [reply]
Yes, it would require samba. But Linux Mint has an easy way to create a share on a Linux Mint system. Simply right-click a folder that you want to share and select "Share Options". You can install Samba, set up users, and config the firewall. They made it very simple for the average user.
